imagedephi/redact/tiff.py

```diff
@@ -3,20 +3,28 @@
 from collections.abc import Generator
 from pathlib import Path
 from typing import TYPE_CHECKING
+import uuid
 
+from PIL import Image
 import tifftools
 import tifftools.constants
 
-from imagedephi.rules import ConcreteMetadataRule, FileFormat, TiffRules
+from imagedephi.rules import (
+    ConcreteImageRule,
+    ConcreteMetadataRule,
+    FileFormat,
+    ImageReplaceRule,
+    TiffRules,
+)
 from imagedephi.utils.tiff import get_tiff_tag
 
 from .redaction_plan import RedactionPlan
 
 if TYPE_CHECKING:
-    from tifftools.tifftools import IFD, TiffInfo
+    from tifftools.tifftools import IFD, TagEntry, TiffInfo
 
 
-class TiffMetadataRedactionPlan(RedactionPlan):
+class TiffRedactionPlan(RedactionPlan):
```

Comment:
Can you split this trivial rename into a separate PR? It'd reduce the scope of this PR and make it easier to review?

Reply:
a49baef

Reply:
Can you make this a PR, so we can merge it ASAP and get it out of the diff for this PR?